translations/README.bg.md

```diff
@@ -0,0 +1,86 @@
+# GitHub Резюме Генератор
+
+![React GitHub Resume](./src/assets/readme/screenshot.png)
+
+## Съдържание
+
+- [Относно](#about)
```

Comment:
Once time you change the topic name, you need to change the links too

Reply:
I looked at the Russian Readme for reference btw and none of this is done there as well :/
I'll adjust it now.

Reply:
Thank you for noticing, I'll see that